pwnlib/rop/rop.py

```diff
@@ -398,6 +413,9 @@ class ROP(object):
     0x0078:             0x2b ss
     0x007c:              0x0 fpstate
 
+.. doctest::
```

Comment:
Why have you broken this into multiple `doctest` blocks?

They are all part of the same example, AND they all have the same requirements.  Separately, `binutils_i386` and `qemu_i386` should not exist at all.  We always assume that Pwntools is running from an `amd64` machine which has binutils installed, and can execute `i386` binaries natively.

`amd64` is the **ONLY** supported architecture to run Pwntools from.

Reply:
It already was multiple doctest blocks. Empty lines are interpreted as a block separator.